sdks/python/apache_beam/transforms/core.py

```diff
@@ -2746,6 +2747,11 @@ def expand(self, pcoll):
     combine_fn = self._combine_fn
     fanout_fn = self._fanout_fn
 
+    if self.isinstance(pcoll.windowfn, SlidingWindows):
```

Comment:
actually, this statement wouldn't work.

Reply:
I'll push a fix.

Reply:
Oh, derp. I didn't notice I had put `self.` in there. Yeesh